pkg/commands/options/build.go

```diff
@@ -67,6 +67,12 @@ type BuildOptions struct {
 
 	// BuildConfigs stores the per-image build config from `.ko.yaml`.
 	BuildConfigs map[string]build.Config
+
+	// ImageConfigs stores the per-image image config from `.ko.yaml`.
+	ImageConfigs map[string]build.ImageConfig `yaml:",omitempty"`
+
+	// DockerRepo stores the global environment variables from `.ko.yaml`.
```

Comment:
This comment is out of date.

Reply:
ah sorry for this, thank you fixed right now